glossar/README.md

```diff
@@ -70,7 +70,7 @@ Datenerfassungsgerät welches elektronische Aufzeichnungen an die mit ihr verbun
 
 ### Kassenseriennummer
 
-Die Kassenseriennummer ist eine Information, die auf den Beleg geprintet werden muss. Dazu wird von fiskaltrust die CashboxIdentification verwendet, Base64 encodiert, Sonderzeichen entfernt  und von der fiskaltrust.Middleware im Signatureblock in der Response (ftSignatureTypes) zurückgeliefert. Die CashboxIdentifikation selbst ist die frei wählbare Bezeichnung für eine Queue. Sie wird auch von der SCU als ClientId für die TSE verwendet. Deshalb ist es wichtig hierbei einen "printable string" mit maximal 20 Zeichen einzugeben. 
+Die Kassenseriennummer ist eine Information, die auf den Beleg geprintet werden muss. Dazu wird von fiskaltrust die CashboxIdentification verwendet, Base64-kodiert, Sonderzeichen entfernt  und von der fiskaltrust.Middleware im Signatureblock in der Response (ftSignatureTypes) zurückgeliefert. Die CashboxIdentifikation selbst ist die frei wählbare Bezeichnung für eine Queue. Sie wird auch von der SCU als ClientId für die TSE verwendet. Deshalb ist es wichtig hierbei einen "printable string" mit maximal 20 Zeichen einzugeben. 
```

Comment:
1. "geprintet" --> "gedruckt"

2. "CashboxIdentification" --> "ftCashboxIdentification"

3. "Signatureblock" in anderem Dokument auch "Signaturen - Block" genannt --vereinheitlichen--> "ftSignatureType"

4. Ist das diese clientid? https://docs.fiskaltrust.cloud/api/fiskaltrust.ifPOS.v1.de.UnregisterClientIdResponse.html?q=clientid

Reply:
danke, wird aktualisiert!